src/Tribe/PUE/Checker.php

```diff
@@ -493,7 +493,7 @@ private function get_api_message( $info ) {
 		 */
 		public function display_license_error_message() {
 			$plugin_info  = $this->json_error;
-			$license_tab = admin_url( 'wp-admin/edit.php?post_type=tribe_events&page=tribe-common' );
+			$license_tab = admin_url( 'edit.php?page=tribe-common&tab=licenses&post_type=tribe_events' );
```

Comment:
It's still there, it's just been moved (to line 520 at this point) :smile: 

That said, it doesn't _seem_ to cause any functional problems so maybe we just roll with it.